tools/migrations/add_icon_name_to_rss_feed.py

```diff
@@ -0,0 +1,12 @@
+import zeeguu.core
```

Comment:
why is this needed?
it is not referenced in the checklist?

Reply:
I think this might have been one of the migrations we deleted, but I still had it tracked in my branch. I think we can remove it if you also agree (or move it to the old folder?)

Reply:
ah. that makes sense. let's delete it then!